molongjoe/CapstoneProject
# Fix for weapon ownership bug

I'm fairly sure this works to fix the bug where a late joiner doesn't see a players' gun if it was picked up before the late joiner joiny joined the jawn.

Also stopped identifying gun owners by their nicknames because that was a bad idea. Now it goes by Player.ActorNumber to identify weapon owner in Room.CustomProperties

To test:
1) build the game
2) Join a game room and pick up a gun
3) Join same game room with second client and check if first client's player is holding the gun. If you see gun, we all good!
